olive/cli/auto_opt.py

```diff
@@ -137,6 +99,34 @@ def register_subcommand(parser: ArgumentParser):
             ),
         )
 
+        # DynamicToFixedShape options
+        sub_parser.add_argument(
+            "--dynamic-to-fixed-shape-dim-param",
+            type=str,
+            nargs="*",
+            default=None,
+            help="Symbolic parameter names to use for dynamic to fixed shape pass.",
+        )
+        sub_parser.add_argument(
+            "--dynamic-to-fixed-shape-dim-value",
+            type=int,
+            nargs="*",
+            default=None,
+            help="Symbolic parameter values to use for dynamic to fixed shape pass.",
+        )
+
+        # MixedPrecisionOverrides options
+        sub_parser.add_argument(
+            "--mixed-precision-overrides-config",
+            type=str,
+            nargs="*",
+            default=None,
+            help=(
+                "Dictionary of name to precision. Has to be even number of entreis with even "
```

Comment:
Is this only for FP16 as precision?

Reply:
Do we need to expose this via CLI?
CLI is not meant to cover all advanced scenarios.

Reply:
I see, they are required. Let's clarify which precision requires the config.